config/routes.rb

```diff
@@ -45,6 +47,9 @@
     patch '/committee_report_export', to: 'reports#committee_report_export', defaults: { format: 'csv' }, as: :committee_report_export
     get '/confidential_hold_report', to: 'reports#confidential_hold_report_index', as: :confidential_hold_report_index
     patch '/confidential_hold_report_export', to: 'reports#confidential_hold_report_export', defaults: { format: 'csv' }, as: :confidential_hold_report_export
+    get '/committee_member_report', to: 'reports#committee_member_report_index', as: :committee_member_report_index
```

Comment:
Is this needed?  I may not be understanding, but doesn't the button on the admin page just download the report (hit the patch action below)?  This index is never used?